data/korath/korath missions.txt

```diff
@@ -257,3 +257,331 @@ mission "Small Scale Delivery"
+			choice
+				`	"I'll go help, then."`
+					accept
+				`	"I won't risk my ship for the Korath."`
+					goto careless
+
+			label careless
+			`	The Quarg is silent for some time. Eventually it responds, "Life is rare and precious across the vastness of space. Look about the Korath worlds and see what remains when one cares not what others suffer."`
+				decline
+
+
+mission "Efreti: Distress Call 2"
+	landing
+	name "Return the <EDC-ship> survivors"
+	description "Transport the <bunks> survivors of the <EDC-ship> to <planet>."
+	source "Seleptra Nak"
+	destination "Laki Nemparu"
+	passengers 40
+	to offer
+		has "Efreti: Distress Call 1: done"
+	on offer
+		conversation
+			`It takes you some time to find the <EDC-ship>. You had expected a beacon of some sort, but either they are keeping radio silence to avoid drawing the Mereti, or there were no survivors. When you manage to open the hatch, you find yourself staring down a thermal rifle.`
+			`	The Korath shouts something at you, and you hold your hands out with your palms forward. He limps back a few steps, but keeps the weapon trained on you. Slowly pulling a communicator out, you play a recording of the Quarg transmission you received earlier. Once the Korath hears the Quarg's voice, he becomes excited and lowers his rifle. "<planet>?" he asks repeatedly. You nod your head and show your palms again.`
+			`	He leads you into the ship, shouting into a communication device that is then broadcasted on speakers throughout the vessel. You can only glance around as you move through the engineering section, but it's clear they are running on emergency power, and several important-looking systems are destroyed. The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree, while more than half are incapacitated or unconscious. Even if the ship is spaceworthy, there aren't enough crew to fly it.`
+			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms, and makes quick circles. You think he is agreeing and wants to hurry. Calling your crew over to help, the survivors are moved onboard the <ship>. Within a couple of hours you are ready to go.`
+				to display
+					"flagship crew" > 1
+				accept
+			# No "to display" is required here due to the "accept" above:
+			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer the survivors to the <ship>.`
```

Comment:
```suggestion
			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms, and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer the survivors to the <ship>.`
```
The comma is there in the version of this line that appears if you have multiple crew, should probably be here too.